server/src/internalClusterTest/java/org/elasticsearch/action/admin/cluster/tasks/ListTasksIT.java

```diff
@@ -128,7 +128,7 @@ public void testWaitForCompletion() throws Exception {
             }));
 
         // briefly fill up the generic pool so that (a) we know the wait has started and (b) we know it's not blocking
-        // flushThreadPool(threadPool, ThreadPool.Names.GENERIC); // TODO it _is_ blocking right now!!, unmute this in #93375
+        flushThreadPool(threadPool, ThreadPool.Names.GENERIC);
```

Comment:
Hm now that I look at this again, with these changes we're not using the `GENERIC` pool any more (indeed there's no forking at all when waiting for completion) so this line doesn't make sense. We can just drop this (and therefore inline `flushThreadPool` at its only other call-site).